code/game/objects/items/weapons/grenades/flashbang.dm

```diff
@@ -34,4 +34,4 @@
 		M.soundbang_act(1, 200, 10, 15)
 
 	else
-		M.soundbang_act(1, max(20/max(1,distance), 3), rand(0, 5))
+		M.soundbang_act(1, max(200/max(1,distance), 60), rand(0, 5))

```

Comment:
@Cyberboss the last time I tried to fix this I made it 20 instead of 200, it should be 200 to be consistent with flash stun times since the old was 10 which is 20 seconds.